apis/duck/v1alpha1/condition_type_test.go

```diff
@@ -225,3 +225,31 @@ func TestJson(t *testing.T) {
 		})
 	}
 }
+
+func TestStatusGetCondition(t *testing.T) {
+	foo := Condition{
```

Comment:
I'd go with `foo := &Condition{}`, and then just `cmp.Equal()` or `cmp.Diff()`.
Alternatively `&foo` below and avoid `got == nil` checks.

Reply:
Done